.env.local.example

```
@@ -26,6 +26,8 @@ NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID=
 ONE_CLICK_URL=https://1click.chaindefuser.com/
 # ⚠️ REQUIRED
 ONE_CLICK_API_KEY=
+# ⚠️ REQUIRED
+ONE_CLICK_DB_URL=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ONE_CLICK_DB_URL key should go before the ONE_CLICK_URL key

(UnorderedKey)